.github/workflows/backend_associated_test_file_check.yml

```diff
@@ -25,22 +25,13 @@ jobs:
           python-version: '3.8.15'
           architecture: 'x64'
       - uses: ./.github/actions/merge
-      - name: Cache node modules and third_party/static
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: |
-            /home/runner/work/oppia/yarn_cache
-            /home/runner/work/oppia/oppia/third_party/static
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock', 'dependencies.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-      - uses: ./.github/actions/install-oppia-dependencies
+      - name: Initialize Containers
+        run: |
+          sudo make build
```

Comment:
Will it build all the containers including angular-build, redis etc? If yes, why do we need to build those images for running `backend_associated_test_file_check`?

Reply:
removed the build step as not required